app/translations/en.json

```diff
@@ -1,10 +1,11 @@
 {
-  "repo_list": "Repository List",
-  "repo_search": "Repository Search",
-  "repo_search_default": "Search for a repository by entering it's name in the search box",
-  "get_repo_details": "Get details of repositories",
-  "search_query": "Search query: {repoName}",
-  "matching_repos": "Total number of matching repos: {totalCount}",
+  "itunes_title": "iTunes store",
+  "songs_search": "itunes song search",
+  "songs_list": "Tracks List",
+  "songs_search_by_user": "Song name you want to listen currently",
+  "search_songs": "Search songs",
+  "search_query": "Search query: {trackName}",
+  "matching_songs": "Total number of matching songs: {totalCount}",
```

Comment:
follow the same naming convention. If you are using are starting with a capital letter use the same convention unless there is a use case where you dont have to